internal/infrastructure/middleware/enforce_identity_test.go

```diff
@@ -482,64 +506,76 @@ func TestEnforceIdentityOrder(t *testing.T) {
 	}
 }
 
+// IdentityAlwaysTrue is a predicate that always return nil
+// so everything was ok.
+// data is the reference to the identity.Identity data.
+// Return nil on success or an error with additional
+// information about the predicate failure.
+func identityAlwaysTrue(*identity.XRHID) error {
```

Comment:
The comment doesn't reflect the implementation. E.g. text `identityAlwaysTrue is a predicate that always return nil as if everything was ok` would be enough.